tests/integration/targets/test_proxysql_mysql_users/tasks/main.yml

```
@@ -7,82 +7,103 @@
 ### tests
 
 - name: "{{ role_name }} | test_create_using_check_mode | test create mysql user using check mode"
-  import_tasks: test_create_using_check_mode.yml
+  include_tasks: test_create_using_check_mode.yml
```

```suggestion
  include_tasks: test_create_using_check_mode.yml
```
could you please not add all unrelated stuff to this single PR? unrelated stuff should go to separate PRs